frontend/public/auth.html

```diff
@@ -3,9 +3,85 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Auth page</title>
+    <link rel="stylesheet" href="/styles.css">
+    <title>Login</title>
 </head>
-<body>
-    <h1>Login/Sign up</h1>
+<body class="login">
+    <table class="loginForm">
+        <tr>
+            <td><label for="usernamefield">Username:</label></td>
+            <td><input type="text" class="loginPageText" id="usernamefield" placeholder="Enter username"></td>
+        </tr>
+        <tr>
+            <td><label for="passwordfield">Password:</label></td>
+            <td><input type="password" class="loginPageText" id="passwordfield" placeholder="Enter password"></td>
```

Comment:
Since we have no idea how to implement passwords, we will not be accepting any. Our login/signup will solely consist of just the username